x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/dashboards/entity_analytics.cy.ts

```diff
@@ -330,21 +330,15 @@ describe('Entity Analytics Dashboard', { tags: ['@ess', '@serverless'] }, () =>
 
     describe('Risk Score enabled but still no data', () => {
       before(() => {
-        cy.task('esArchiverLoad', { archiveName: 'risk_scores_new_no_data' });
```

Comment:
I noticed that many risk engine tests are showing an error when enabling the engine because the index was already created. So, I am updating the tests to enable the risk engine before loading data with the archiver.